run.js

```diff
@@ -125,7 +125,7 @@ if (!module.parent) {
       const result = await run(code, process.env.JSEVAL_ENV || 'node-cjs', +process.env.JSEVAL_TIMEOUT || undefined);
       process.stdout.write(inspect(result, {
         depth: process.env.JSEVAL_DEPTH ? +process.env.JSEVAL_DEPTH : undefined,
-        compact: process.env.JSEVAL_COMPACT === 'false' || process.env.JSEVAL_COMPACT === '0' ? false : process.env.JSEVAL_COMPACT
+        compact: process.env.JSEVAL_COMPACT === 'false' ? false : process.env.JSEVAL_COMPACT
```

Comment:
make this `!!process.env.JSEVAL_COMPACT`... it should be off by default.

Reply:
The tests don't pass with `!!process.env.JSEVAL_COMPACT`, but they do with  `process.env.JSEVAL_COMPACT` as just committed